include/CircuitSimulator.hpp

```diff
@@ -12,60 +12,61 @@
 #include <string>
 
 struct ApproximationInfo {
-    enum ApproximationWhen {
+    enum ApproximationMethod {
```

Comment:
`ApproximationMethod` seems like a misleading name (especially if we want to introduce removing edges to the approximation. How about `ApproximationDriver` oder `ApproximationStrategy` (in analogy to the terms in the paper)?

Reply:
I like approximation strategy. Will adapt correspondingly.

Reply:
Done in https://github.com/cda-tum/ddsim/pull/186/commits/3ae652bcabcb016ae18e19b9e65a198140d6d0d0